Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'fpm'
```

Comment:
The packaging is done via `fpm`. But since nothing here does invokes `bundle install` it's superfluous. I'm happy to pull it.